README.rst

```diff
@@ -66,6 +42,26 @@ Why use coala?
   without you having to worry.
 - *Unified interface*: One tool for all programming languages.
 
+If you want to learn more about coala, its functionality and its usage,
+please take a look at our
+`tutorial <http://coala.rtfd.org/en/latest/Users/Tutorials/Tutorial.html>`__.
+
+Installation
+------------
+
+coala can be installed with ``pip install coala``. If you need more
```

Comment:
Are you sure you don't want to say `pip3 install coala` instead? Python 2.x is more popular and on such machines `pip` defaults to 2.x. (Or at least mention it's Python3, but that will be implied with `pip3`). I know that an error message is thrown, but it *might* scare away some who don't have much experience with pip and stuff.